### ISSUE TITLE
How to connect the exporter to multiple instance name

### ISSUE THREAD

**Issue by Chhith02** [iamseth/oracledb_exporter#203]
Hello,

I'm following the notes for the multiple host instance and it works perfectly. But i'm facing an issue with a DB server which have multiple service name; I don't know how to set the environment variable DATA_SOURCE_NAME in this case, I can only set a single service name.

Here's an example : 
This is the looks of the tnsnames.ora file :

database =
(DESCRIPTION =
  (ADDRESS_LIST =
    (ADDRESS = (PROTOCOL = TCP)(HOST = apophis.db)(PORT = 1520))
    (ADDRESS = (PROTOCOL = TCP)(HOST = apophis.db)(PORT = 1525))
  )
  (CONNECT_DATA =
    (SERVICE_NAME = DBTHOR_01)
    (SERVICE_NAME = DBLOKI_01)
  )
)

And how I set the DATA_SOURCE_NAME variable : 
export DATA_SOURCE_NAME=user/password@DBTHOR_01 or export DATA_SOURCE_NAME=user/password@DBLOKI_01

I'm trying this but It didn't work : 
export DATA_SOURCE_NAME="user/password@DBTHOR_01,DBLOKI_01"

So i'm asking myself if is it possible to set multiple service name for the exporter ?

Thanks for your help

Kinds

**Comment by Yannig** [comment]
No, this is not possible. You need to launch two different exporters ...

**Comment by tianamina** [comment]
1 service = 1 exporter, as we do for our multiples oracle instances on our different servers
